### ISSUE TITLE
Delete digital identity

### ISSUE THREAD

**Issue by veitmo** [amosproj/amos2022ss04-digital-identity#12]
## User story
1. As a {HR employee}
2. I want / need { to be able to delete an existing digital identity from the system by putting it into a bin-section. The deleted digital identities should still be visible in a dedicated bin-section}
3. So that {employees which have left the company also loose their digital identity-rights}

## Acceptance criteria
* button on frontend for deletion
* bin-section 
* ...

## Definition of done (DoD)
* Added only after week 5
* The same for all features
* Here goes the project specific part

## DoD general criteria
* Feature has been fully implemented
* Feature has been merged into the mainline
* All acceptance criteria were met
* Product owner approved features
* All tests are passing
* Developers agreed to release
